src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/DigitransitRealtimeStopPropertyMapper.java

```diff
@@ -32,10 +34,19 @@ protected Collection<KeyValue> map(RegularStop stop) {
       .stream()
       .anyMatch(alert -> alert.noServiceAt(currentTime));
 
+    var serviceDate = LocalDate.now(transitService.getTimeZone());
+    boolean stopTimesExist = transitService
+      .getStopTimesForStop(stop, serviceDate, ArrivalDeparture.BOTH, true)
```

Comment:
I tried to quickly take a look if this also includes stop times for patterns that have been added through an realtime update but I'm not sure. However, if it doesn't, I think we have plenty of other api calls which are also broken in that sense so maybe we shouldn't care about it for now.

Reply:
It seems like the realtimes are added as the stopTimesForStop fetches the patterns with includeRealtimeUpdates set as true.